CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 * Fix jsfiddle reference.
 
+### Changed
+* [Breaking] Property accessors are no longer wrapped when they already exist. Instead the `noAccessor` flag should be set when a user defined accessor is created. ([#450](https://github.com/Polymer/lit-element/issues/450)).
```

Comment:
I think you can remove the "`noAccessor` flag should be set" part

Reply:
Added content @kevinpschaaf suggested.